mobile/lib/shared/cache/custom_image_cache.dart

```diff
@@ -1,6 +1,5 @@
 import 'package:flutter/painting.dart';
-
-import 'original_image_provider.dart';
+import 'package:immich_mobile/modules/asset_viewer/image_providers/original_image_provider.dart';
```

Comment:
This file should use the new local image provider. Otherwise, the both large and small images will end up in the same cache again.

Reply:
Thank you! I've removed the reference.